pages/update-password.tsx

```diff
@@ -0,0 +1,125 @@
+import { inferSSRProps } from "@/lib/inferSSRProps";
```

Comment:
Spacing and indentation should be consistent throughout the codebase. We use the 2-space indentation I think.

Reply:
Ok, I run `npm run format` I think this is enough